submissions/Complexity Corner - The Mall/Complexity Corner - The Mall.md

```diff
@@ -0,0 +1,105 @@
+
+[Integrated mall example: The "Jump Start Base" by Nilaus]
+
+[Integrated mall example: My TINS starter base]
+
+A consequence of the integrated design is your mall machines interrupting your science production as they resupply their outputs, which may be something you prefer already if you want research and factory building to happen in parallel. These interruptions are most severe upon first building the mall and are much less later on, because many of the mall machines tend to be idle more often than not. A disadvantage of improvising an integrated design is that you can run low on space and spaghettify pretty quickly.
+
+The alternative approach is building a dedicated area for your mall. Without relying on your science production, a dedicated mall requires its own resource supply logistics, which can be belts branching off a main bus or dedicated resupply trains/bots. Alternatively the area can have its own intermediate item production area, perhaps even including its own furnaces such that it only needs ores. A dedicated mall area is easier to lay out and expand compared to an integrated area, especially when improvising. This approach also makes it easier to control how much of your resources you want to divert to science production versus mall production.
+
+[Dedicated mall example from community]
+
+## **Which recipes should be included?**
+
+The ideal mall would include a machine for practically everything, but it would be huge and resource-intensive. It makes more sense to build according to what you really need. For a starter mall, I would recommend including the first-tier items you will need all the time when building the factory: assembling machines, inserters, belts, underground belts, and splitters. The beautiful thing about all of these items is that they are ingredients for their next tiers. As a result, overproducing them is rarely wasteful and you can easily handcraft second-tier items when you have a decent supply of them and intermediate items. For example, with splitters and intermediates in hand, a fast splitter takes only two seconds to craft instead of nearly 30 seconds when using raw materials. Other items that are great to pick up from an early-game mall are gears, electronic circuits, long-handed inserters, pipes, firearm magazines, gun turrets, and power poles. If you want to plan in advance for the mid game, it would be useful for your starter mall to have empty machines reserved for making engines (for vehicles and pumps) and the most common second-tier items: assembling machine 2's, fast inserters, and fast belts..
```

Comment:
Does it seem you are from now on only discussing dedicated malls?

Reply:
Good idea to add: for integrated malls, the recipe to include simply depends on what is available to you in each area